resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java

```diff
@@ -54,7 +57,11 @@ private CircuitBreakerMetrics(String prefix, Iterable<CircuitBreaker> circuitBre
         circuitBreakers.forEach(circuitBreaker -> {
                 String name = circuitBreaker.getName();
                 CircuitBreaker.Metrics metrics = circuitBreaker.getMetrics();
-
+                //add a metric that reports the status and a boolean on the open state
```

Comment:
I'm afraid that this metric can be really misleading. If your metrics reporting period is bigger then `CircuitBreakerConfig.waitDurationInOpenState` there will be situations when metrics wont show you actual `OPEN` state.

Reply:
Would monitoring the opposite be better? have a "is_closed" metric?